config.lua.dist

```diff
@@ -176,6 +176,7 @@ serverSaveNotifyDuration = 5
 serverSaveCleanMap = false
 serverSaveClose = false
 serverSaveShutdown = true
+serverSaveTime = "06:00"
```

Comment:
Shouldn't it be 06:00:00 to follow the same pattern from the data/scripts/globalevents/server_save.lua file?